packages/gatsby-remark-autolink-headers/src/gatsby-browser.js

```diff
@@ -11,8 +11,15 @@ const getTargetOffset = hash => {
         document.body.scrollTop
       let clientTop =
         document.documentElement.clientTop || document.body.clientTop || 0
+      let scrollMarginTop = window
```

Comment:
`scroll-margin-top` isn't supported in safari. `scroll-snap-margin-top` is, so we should test both values. We should also set the default to 0 for browsers who don't support it like IE11

```suggestion
      const scrollMarginTop = window
        .getComputedStyle($0)
        .getPropertyValue("scroll-margin-top") || window
        .getComputedStyle($0)
        .getPropertyValue("scroll-snap-margin-top") || 0
```

Reply:
Cheers, that's a nice catch! I've added the safari safeguard you suggested and fixed some linting issues 🙌 

The const/let is a little all over the place in the file, just let me know if you want to move it in any specific direction

Reply:
i'm fine with either 👍